modules/mesh-layers/src/scenegraph-layer/gltf-utils.js

```diff
@@ -5,7 +5,7 @@ export async function waitForGLTFAssets(gltfObjects) {
 
   gltfObjects.scenes.forEach(scene => {
     scene.traverse(model => {
-      Object.values(model.model.program.uniforms).forEach(uniform => {
+      Object.values(model.model.uniforms).forEach(uniform => {
```

Comment:
Shall we use `getUniforms` in case of future changes?

Reply:
That's a good idea.